### ISSUE TITLE
FlutterFlow stuck on loading screen

### ISSUE THREAD

**Issue by gabrielguerin** [FlutterFlow/flutterflow-issues#31]
## Expected behavior
FlutterFlow launches

## Current behavior
FlutterFlow gets stuck on loading screen and never launches

## To Reproduce
1. Launch Chrome, Edge or Firefox in normal or incognito mode
2. Sign in to FlutterFlow account
3. Launch any FlutterFlow app
4. Observe that FlutterFlow doesn’t launch

## Context (required)
I just created my account as I wanted to try FlutterFlow. However, whenever I try to launch FlutterFlow, it stays stuck on the loading screen. I have tried using Firefox, Edge and Chrome, each in incognito and normal modes. Inspecting the browser gives me the following errors:
- `Uncaught TypeError: Cannot read properties of null (reading 'b')`
- `POST [https://us-central1-flutterflow-io-6f20.cloudfunctions.net/getCountryCode 500](https://us-central1-flutterflow-io-6f20.cloudfunctions.net/getCountryCode%20500)` 
- `POST [https://us-central1-flutterflow-io-6f20.cloudfunctions.net/getCountryCode 500](https://us-central1-flutterflow-io-6f20.cloudfunctions.net/getCountryCode%20500)`

Here is what I have tried so far:
- Log in and out of FlutterFlow account from my [Project Dashboard](https://app.flutterflow.io/). 
- Clear my browsers' cookies and cache.
- Upgrade to the latest version of FlutterFlow by logging in to FlutterFlow and selecting Ctrl/Cmd + R.
- Use a different browser.

I could not try using a different computer, as I do not have one,

## Screenshots / recordings
https://user-images.githubusercontent.com/39315283/194834801-f3b8279e-be8a-4650-92f7-59146eb8a6e5.mp4

## Your environment
* Platform: Web 
* Browser name and version: Firefox 105.0.3 (64-bit), Chrome 105.0.5195.127 (64-bit) & Edge 106.0.1370.37
* Operating system and version: Windows 11 Home 22H2

**Comment by wenkaifan0720** [comment]
I am unable to reproduce this error. Seems like the problem is with Firebase itself? 

It is weird that the user is able to launch the dashboard page but unable to go into the projects. The error seems to suggest there is something wrong with the country code. But I can't find anything related to country code when navigating to a project.

**Comment by agreaves** [comment]
This was a very bizarre issue, but I believe this should be fixed now. Please try logging out, refreshing the page, and logging in again. Hopefully it should work at that point.

**Comment by gabrielguerin** [comment]
For the record, I had a friend create an account and try to launch the application and he had the same problem. But in any case, the FlutterFlow support answered me, and asked me to try again, and the problem somehow got solved. I will close this issue.

**Comment by SirSpammenot** [comment]
This sounds VERY similar to my issue. Hrmmm. Glad to help troubleshoot some if that helps.  

My laptop seems to be working correctly, and I was able to reset (ie: set for the first time) my password.  Then using the SAME email address, but NOT using SSO, FF seemed to be logging in normally.  I was able to see and load my projects at least, but then neither of the project's API calls worked.  so still something messed up..

Before setting a password: 
I got a little further also using a custom domain with google, I can SSO into the product, but then am having issues loading projects. I have traced the actual error to DNS but cannot explain why nothing else seems to be effected.  Also cannot explain why FF seemed to work OK for the first few months.

In this image I was trying to reset the password.  Note the error message at the bottom, and the console DNS resolve issues...
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/3230925/511a7beb-8329-43bc-9a68-f601a343c6b7)

Similar to the Original trouble report, I can no longer even list my projects!  Weird.
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/3230925/1175875f-e026-40ab-b855-ac33eda9f083)
